### ISSUE TITLE
NOTETHIS

### ISSUE THREAD

**Issue by neagle** [neagle/gokibitz#70]
https://twitter.com/Kinch_ahoy/status/593488368090161152

Want to mark specific moves with "NOTETHIS" in chat during the game. Then quickly jump to them in review. Interesting feature?

**Comment by bengozen** [comment]
I could definitely see this being valuable to players. The only thing I would be concerned with is how it impacts the game playing dynamic. Maybe it's not as big a deal as I'm thinking, but perhaps typing "NOTETHIS" in the chat every few moves might get annoying to the opponent? 

To reiterate though, it's a +1 from me with some reservations.

**Comment by neagle** [comment]
Yeah, that's a valid point. I'm a big fan of study tools, which is why I like this, but its actual application to game play might need some thinking about.

OGS already has a tool like this -- they've got built-in Malkovich mode, where you can take notes during the game that aren't visible to your opponent as you play.

**Comment by neagle** [comment]
It *might* make sense to think about simply having a way to navigate between game moves with game comments on them, much like the feature we just added to GoKibitz that lets you navigate between game moves with user comments on them.

**Comment by bengozen** [comment]
That would be an interesting feature, but I wonder if the value of that feature would be diminished if you had a game where you had friends watching and kibitzing. Like you mentioned, this is less of an issue with options that have Malkovich mode, but this is definitely a UX thing that I think we'll have to wrangle with with live online play.
